components/blitz/src/omero/gateway/Connector.java

```diff
@@ -855,7 +855,9 @@ public Connector call() throws Exception {
                 p.name = userName;
                 p.eventType = "Sessions";
                 ISessionPrx prx = entryEncrypted.getSessionService();
-                Session session = prx.createSessionWithTimeout(p, 0L);
+                Session session = prx.getSession(secureClient.getSessionId());
+                long timeout = session.getTimeToIdle().getValue();
+                session = prx.createSessionWithTimeout(p, timeout);
```

Comment:
This method apparently sets the `timeToLive` and you are setting the `timeToIdle` from the first session, you might want to use [createSessionWithTimeouts(principal, long, long)](http://downloads.openmicroscopy.org/omero/5.1.3/api/ome/services/sessions/SessionBean.html#createSessionWithTimeouts(ome.system.Principal,%20long,%20long)) instead.